src/Copy/BrandCopy.elm

```diff
@@ -642,7 +642,7 @@ brandCopy key =
                 it was being logged."""
 
         StoryThree3ImageAlt ->
-            CopyText "✨"
+            CopyText "Illustration of a woman talking to a man taking notes and listening to her with two boats in the background, one titled apology and the other complaint"
```

Comment:
Maybe "Illustration of a shrugging woman, talking to a man who is taking notes and listening to her with two boats in the background, one titled apology and the other complaint"